src/pages/projects/{MarkdownRemark.parent__(File)__name}.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { graphql } from 'gatsby'
+
+const Project = ({ data }) => {
+  const title = data.markdownRemark.title
+  const html = data.markdownRemark.html
+  return (
+    <div>
+      <h1>{title}</h1>
+      <div dangerouslySetInnerHTML={{ __html: html }}></div>
+    </div>
+  )
+}
+
+export default Project
+
+export const query = graphql`
+  query PortfolioListQuery($id: String) {
+    markdownRemark(id: { eq: $id }) {
+      id
+      html
+      frontmatter {
```

Comment:
Looks like images still aren't showing up on my end. Let me know if you need help pulling them from the CMS.

Reply:
I imagine we'll need to merge @jscarpelli3 's PR for this to work. Once he finishes up with his ticket you can merge with `main` and we'll see if the images show up.